token/program-2022/src/extension/transfer_fee/processor.rs

```diff
@@ -48,6 +50,52 @@ fn process_initialize_transfer_fee_config(
     Ok(())
 }
 
+fn process_set_transfer_fee(
+    program_id: &Pubkey,
+    accounts: &[AccountInfo],
+    transfer_fee_basis_points: u16,
+    maximum_fee: u64,
+) -> ProgramResult {
+    let account_info_iter = &mut accounts.iter();
+    let mint_account_info = next_account_info(account_info_iter)?;
+    let authority_info = next_account_info(account_info_iter)?;
+    let authority_info_data_len = authority_info.data_len();
+
+    let mut mint_data = mint_account_info.data.borrow_mut();
+    let mut mint = StateWithExtensionsMut::<Mint>::unpack(&mut mint_data)?;
+    let extension = mint.get_extension_mut::<TransferFeeConfig>()?;
+
+    let transfer_fee_config_authority =
+        Option::<Pubkey>::from(extension.transfer_fee_config_authority)
+            .ok_or(TokenError::NoAuthorityExists)?;
+    Processor::validate_owner(
+        program_id,
+        &transfer_fee_config_authority,
+        authority_info,
+        authority_info_data_len,
+        account_info_iter.as_slice(),
+    )?;
+
+    // When setting the transfer fee, we have a few situations:
```

Comment:
```suggestion
    // When setting the transfer fee, we have two situations:
```

Reply:
Hehe when I started writing it I thought there would be more